data/human/free worlds 3 reconciliation.txt

```diff
@@ -1038,9 +1038,9 @@ mission "FW Pug 2"
 	to offer
 		has "FW Pug 1: done"
 	passengers 1
-	blocked `As you are landing, Alondo contacts you and says, "Do you have a bunk free for me?" You do not, so you will need to free one up in order to continue this mission.`
+	blocked `As you are landing, Alondo contacts you and asks, "Do you have a bunk free for me?" You do not, so you will need to free one up in order to continue this mission.`
 	on fail
-		dialog `You have failed an essential Free Worlds mission. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`
+		dialog `You have failed an essential Free Worlds mission. If you want to complete the storyline, revert to the autosave or another earlier snapshot of the game.`
```

Comment:
The only place in the game where "storyline" appears as one word is one person ship hail.
And, every other time this dialog is given, it says "story line".
```suggestion
		dialog `You have failed an essential Free Worlds mission. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`
```